wtforms_appengine/ndb.py

```diff
@@ -358,6 +360,9 @@ def convert_KeyProperty(self, model, prop, kwargs):
         else:
             return KeyPropertyField(**kwargs)
 
+    def convert__ClassKeyProperty(self, model, prop, kwargs):
```

Comment:
Well, that's what I mean. You should do a release on Pypi! :) Right now if you install va Pypi, you get an old (and broken) version.